### PR TITLE
Update bttc testnet info

### PR DESCRIPTION
**Pull request by xq-lu**
This PR includes the following changes:
1. set old chainid 1028 to deprecated. 
2. add new testnet chainid 1029 and update some chain info like RPC URL, and faucets URL.

### CODE REVIEW

**Comment by xq-lu** [comment]
@ligi Please merge, thank you very much.

File: _data/chains/eip155-1029.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "decimals": 18
+  },
+  "infoURL": "https://bt.io/",
+  "shortName": "tbtt",
+  "chainId": 1029,
+  "networkId": 1029,
+  "explorers": [
+    {
+      "name": "Bttc Scan Testnet",
+      "url": "https://testnet.bttcscan.com/",
```
Comment (ligi) [inline _data/chains/eip155-1029.json:18]:
```suggestion
      "url": "https://testnet.bttcscan.com",
```

File: _data/chains/eip155-1029.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      "standard": "EIP3091"
+    },
+    {
+      "name": "Bttc Scan Donau Testnet",
+      "url": "https://testscan.bt.io/",
```
Comment (ligi) [inline _data/chains/eip155-1029.json:23]:
```suggestion
      "url": "https://testscan.bt.io",
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Review by ligi — CHANGES_REQUESTED** [review]
NameMustBeUnique:  name must be unique - but found `BITTORRENTCHAINTESTNET` more than once

also please run the prettier

**Comment by xq-lu** [comment]
> NameMustBeUnique: name must be unique - but found `BITTORRENTCHAINTESTNET` more than once
> 
> also please run the prettier

hi,ligi.already fixed the format errors.

**Comment by xq-lu** [comment]
hi, @Tesbarmix1 thank you .but the status of this PR is still open. I guess it still needs a maintainer to approve running workflows. could you help?

**Comment by xq-lu** [comment]
hi, @Tesbarmix1 , could you merge this pull request ? thank you

**Review by ligi — CHANGES_REQUESTED** [review]
short name must be unique - but found `TBTT` more than once

**Pull request closed without merging**